chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsReceivedIntegrationTest.kt

```diff
@@ -87,7 +87,7 @@ internal object JoinedRoomsReceivedIntegrationTest : Spek({
             )
 
             beforeEachTest {
-                userSubscriptionDispatcher.onEvent(event)
+                userSubscriptionDispatcher.onEvent(event.asElementsEvent())
```

Comment:
yeah that's definitely not nice

Reply:
Any better automagic way of converting them gladly accepted! :)

Reply:
it's fine, my feedback was just stemming from the belief that we shouldn't need that raw type, thanks for enlightening me and keeping the persistence in mind!

Reply:
I think I persisted it in some kind of archive and forgot about it ;)